tests/foundry/common/utils/string-tree.mjs.test-d.ts

```diff
@@ -0,0 +1,8 @@
+import { expectTypeOf } from "vitest";
+
+const s = new foundry.utils.StringTree();
+const node = {} as foundry.utils.StringTree.StringTreeNode;
```

Comment:
Nit; I usually write something like `declare const node: foundry.utils.StringTree.StringTreeNode;`. It avoids the `as` cast which slightly muddles the intent of what's going on here.